tests/test_api_circuit_breaker.py

```diff
@@ -50,6 +56,10 @@ def test_circuit_breaker_500(fake_all_blocks, breaker_test):
         resp = response.json()
         assert all(b['type'] != "wikipedia" for b in resp['blocks'])
 
+        """
+        After the timeout the circuit should be half-open.
+        So one more external call should be done.
+        """
         sleep(3)
```

Comment:
if the reset timeout is 1, why wait 3 seconds ?

Reply:
just to be sure^^